A55-xds-stateful-session-affinity.md

```diff
@@ -421,14 +427,17 @@ or `ring_hash_experimental`) is created as the child policy of
 To propagate [`common_lb_config.override_host_status`][or-host-status] the
 following changes are required:
 
-* the [`XdsClient`][grpc-client-arch] to parse
+* CDS update message will be modified to include the `override_host_status`
```

Comment:
Let's combine this bullet with the next one:

"""
The [`XdsClient`][grpc-client-arch] will parse the [`common_lb_config.override_host_status`][or-host-status] field from the `Cluster` resource and copy the value to a new `override_host_status` field in the CDS update that it sends to the watchers.
"""

Reply:
done